bundles/org.openhab.automation.scriptenginefactory.jython/src/main/feature/feature.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.automation.scriptenginefactory.jython-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+    <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+
+    <feature name="openhab-misc-scriptenginefactory-jython" description="Jython ScriptEngineFactory" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
+        <feature>openhab-core-automation</feature>
```

Comment:
To solve the build issue we have to change this feature to:

```suggestion
        <feature>openhab-core-automation-module-script</feature>
```

ftr: `openhab-core-automation-module-script` depends on `openhab-core-automation` (see https://github.com/openhab/openhab-core/blob/173c93081dd6f7b15047c50c316a0e0bf99b6100/features/karaf/openhab-core/src/main/feature/feature.xml#L82-L86).

Reply:
Very interesting... thank you! Done!